src/main/java/org/breedinginsight/brapi/v2/services/BrAPITrialService.java

```diff
@@ -54,7 +55,9 @@ public class BrAPITrialService {
     private final BrAPITrialDAO trialDAO;
     private final BrAPIObservationDAO observationDAO;
     private final BrAPIListDAO listDAO;
-    private final BrAPIObservationVariableDAO obsVarDAO;
+//    private final BrAPIObservationVariableDAO obsVarDAO;
+
+    private final TraitService traitService;
```

Comment:
vestigial comments left after substituting TraitService for BrAPIObservationVariableDAo should be removed

Reply:
I'll fix this.